octobot_trading/personal_data/transactions/types/realised_pnl_transaction.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, exchange_name, creation_time, transaction_type, currency, sym
         self.trigger_source = trigger_source
         self.side = side
         self.transaction_id = f"{self.exchange_name}" \
-                              f"-{str(uuid.uuid4())[:4]}" \
+                              f"-{str(uuid.uuid4())}" \
```

Comment:
Why?

Reply:
This is sometimes generating duplicate transaction issues in backtestings when multiple orders are filled at the same time

Reply:
Is there a reason for keeping only the last 4 digits there or it's just to save space ?

Reply:
I don't remember why only the last $